package.json

```diff
@@ -16,7 +16,8 @@
     "build-storybook": "storybook build",
     "test": "vitest run",
     "test:coverage": "vitest run --coverage",
-    "test:watch": "vitest"
+    "test:watch": "vitest",
+    "build:tokens": "node  ./scripts/tokenizer.mjs"
```

Comment:
Nice approach, add this script to the build system like this 
`build: "... && yarn build:tokens"`
This will give us the guarantees that the token will be created correctly for every build process